packages/lb-annotation/src/core/toolOperation/rectOperation.ts

```diff
@@ -656,15 +656,15 @@ class RectOperation extends BasicToolOperation {
         const basicWidth = this.basicResult.width * this.zoom;
         const basicHeight = this.basicResult.height * this.zoom;
 
-        if (
-          this.dragInfo.dragTarget !== EDragTarget.Plane &&
-          (selectedRect.x < basicX - 0.01 ||
-            selectedRect.y < basicY - 0.01 ||
-            selectedRect.width > basicX + basicWidth - selectedRect.x + 0.01 ||
-            selectedRect.height > basicY + basicHeight - selectedRect.y + 0.01)
-        ) {
-          return;
-        }
+        // if (
+        //   this.dragInfo.dragTarget !== EDragTarget.Plane &&
+        //   (selectedRect.x < basicX - 0.01 ||
+        //     selectedRect.y < basicY - 0.01 ||
+        //     selectedRect.width > basicX + basicWidth - selectedRect.x + 0.01 ||
+        //     selectedRect.height > basicY + basicHeight - selectedRect.y + 0.01)
+        // ) {
+        //   return;
+        // }
```

Comment:
How can you to ensure the rect is in `drawOutsideTarget `?

Reply:
![image](https://user-images.githubusercontent.com/22559022/236723944-24d66178-8436-4827-bacc-78e393c4abf1.png)